seismicpro/refractor_velocity/refractor_velocity_field.py

```diff
@@ -150,10 +152,41 @@ def __str__(self):
 
         return msg
 
+    @staticmethod
+    def _fit_refractor_velocities(rv_kwargs_list):
+        """Fit a separate near-surface velocity model by offsets and times of first breaks for each set of parameters
+        defined in `rv_kwargs_list`. This is a helper function and is defined as a `staticmethod` only to be picklable
+        so that it can be passed to `ProcessPoolExecutor.submit`."""
+        return [RefractorVelocity.from_first_breaks(**rv_kwargs) for rv_kwargs in rv_kwargs_list]
+
+    @classmethod
+    def _fit_refractor_velocities_parallel(cls, rv_kwargs_list, chunk_size=250, n_workers=None, bar=True, desc=None):
+        """Fit a separate near-surface velocity model by offsets and times of first breaks for each set of parameters
+        defined in `rv_kwargs_list`. Velocity model fitting is performed in parallel processes in chunks of size no
+        more than `chunk_size`."""
+        n_velocities = len(rv_kwargs_list)
+        n_chunks, mod = divmod(n_velocities, chunk_size)
+        if mod:
+            n_chunks += 1
+        if n_workers is None:
+            n_workers = os.cpu_count()
+        n_workers = min(n_chunks, n_workers)
+        executor_class = ForPoolExecutor if n_workers == 1 else ProcessPoolExecutor
+
+        futures = []
+        with tqdm(total=n_velocities, desc=desc, disable=not bar) as pbar:
```

Comment:
```suggestion
        with tqdm(total=n_velocities, desc="Velocity models estimated", disable=not bar) as pbar:
```
It is not possible to change the `desc` using func parameters, so there is no need to pass it from another function,